tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -138,11 +143,18 @@ public void processNe(SourceFeature sf, FeatureCollector features) {
         // since all these are locality, we hard code kindRank to 2 (needs to match OSM section below)
         .setSortKey(getSortKey(minZoom, 2, populationRank, population, sf.getString("name")));
 
-      var script = Script.getScript(sf.getTag("name").toString());
+      String name = sf.getTag("name").toString();
+      var script = Script.getScript(name);
+
       if (!script.equals("Latin") && !script.equals("Generic")) {
         feat.setAttr("pmap:script", script);
+        if (fontRegistry != null && fontRegistry.getScripts().contains(script)) {
+          name = TextEngine.encodeRegisteredScripts(name, fontRegistry);
+        }
       }
 
+      feat.setAttr("name", name);
```

Comment:
Do you think we should leave the proper Unicode in `name` and then have logic in the style called name:mm:pgf` with the non-Unicode encoding? That would make the tileset more consistent for non-MapLibre use cases

Reply:
It is a good idea to keep the original Unicode string in `name` and `name:*`. I prefixed the encoded strings with `pmap:pgf:`.

Examples: 
* `name` contains original Unicode
* `pmap:pgf:name` contains parts that are in a registered script in the encoded string
* `name:ne` contains original Unicode
* `pmap:pgf:name:ne` contains parts that are in a registered script in the encoded string